woocommerce/woocommerce-android
# Jetpack CP: UI for Install Jetpack from App

## Description

This is the first part for the "Jetpack CP: Install Jetpack from app" feature: https://github.com/woocommerce/woocommerce-android/issues/4865#issuecomment-956128458

This PR adds the UI elements and navigation related to the functionality. There is no functionality yet and the intent of this PR is just to check the design and test the user interface.

**Exception:** The circular loading animation design is not yet decided, so for now it is using a temporary animation and it can be ignored.

## To test:
1. Have a Jetpack CP site (e.g: start a .org site with WooCommerce, then install WooCommerce Payments plugin, then set it up to connect to your WordPress.com site),
2. Log in to Woo app and select the Jetpack CP site,
3. Find the "Get the full experience with Jetpack" banner on "My Store" tab, then tap it,
4. This will show the Jetpack benefits dialog. Tap "Install Jetpack" button,
5. This will show the install start dialog. Check the design and test it on different orientations,
6. After checking is done, tap "Install Jetpack" button,
7. This will show the install progress dialog. Check the design and test it on different orientations.
8. After checking is done, tap "Done" and make sure it returns to "My Store" screen.

**Light mode**
| Install start | Install progress |
|-|-|
|<img src="https://user-images.githubusercontent.com/266376/140496008-4a948d9b-98ec-4f00-9094-ed6873cb5104.png" width="250px" />|<img src="https://user-images.githubusercontent.com/266376/140495957-4875972f-c958-477c-b76f-327629185a33.png" width="250px" />|



**Dark mode**
| Install start | Install progress |
|-|-|
|<img src="https://user-images.githubusercontent.com/266376/140503301-b866cc2d-6a6d-4c7b-8aca-c6346fb554bb.png" width="250px" />|<img src="https://user-images.githubusercontent.com/266376/140495777-7f54429e-dc91-45f5-8e70-876fa1560b70.png" width="250px" />|
